docs/tecnico/arquitectura.md

```
@@ -29,6 +29,31 @@ Sales Invoice (submit)
   → timbrado_api.py → FacturAPI.io → SAT
 ```
 
+## Flujo E-Receipt / Autofactura
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)